packages/build/src/barque.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { Barque } from './barque';
+import { expect } from 'chai';
+import Config from './config';
+import sinon from 'sinon';
+
+describe('Barque', () => {
+  let barque: Barque;
+  let config: Config;
+
+  beforeEach(() => {
+    config = {
+      version: 'version',
+      bundleId: 'bundleId',
+      input: 'input',
+      execInput: 'execInput',
+      outputDir: 'outputDir',
+      analyticsConfig: 'analyticsConfig',
+      project: 'project',
+      revision: 'revision',
+      branch: 'branch',
+      evgAwsKey: 'evgAwsKey',
+      evgAwsSecret: 'evgAwsSecret',
+      downloadCenterAwsKey: 'downloadCenterAwsKey',
+      downloadCenterAwsSecret: 'downloadCenterAwsSecret',
+      githubToken: 'githubToken',
+      segmentKey: 'segmentKey',
+      rootDir: '../../../',
+      appleUser: 'appleUser',
+      applePassword: 'applePassword',
+      appleAppIdentity: 'appleAppIdentity',
+      isCi: true,
+      platform: 'linux',
+      buildVariant: 'linux',
+      repo: {
+        owner: 'owner',
+        repo: 'repo',
+      }
+    };
+
+    barque = new Barque(config);
+  });
+
+  describe('.releaseToBarque', () => {
+    context('platform is linux', () => {
+      it('execCurator function succeeds', async() => {
+        barque.execCurator = sinon.stub().returns(Promise.resolve(true));
+        const tarballURL = 'https://s3.amazonaws.com/mciuploads/mongosh/5ed7ee5d8683818eb28d9d3b5c65837cde4a08f5/mongosh-0.1.0-linux.tgz';
+
+        let err;
+
+        try {
+          await barque.releaseToBarque(tarballURL);
+        } catch (error) {
+          err = error;
+        }
+        expect(err).to.be.undefined;
+        expect(barque.execCurator).to.have.been.called;
+      });
+
+      it('execCurator function fails', async() => {
+        barque.execCurator = sinon.stub().returns(Promise.reject(new Error('error')));
```

Comment:
I think we should be able to write just `sinon.stub().rejects(new Error('error'))`